src/app/components/schedule/schedule.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit } from '@angular/core';
+import { Trip } from 'src/app/models/trip';
+import {ScheduleService} from 'src/app/services/schedule-service/schedule.service';
+import {Observable} from 'rxjs';
+import { AuthService } from 'src/app/services/auth-service/auth.service';
+import { RouterModule } from '@angular/router';
+
+
+@Component({
+  selector: 'app-schedule',
+  templateUrl: './schedule.component.html',
+  styleUrls: ['./schedule.component.css']
+})
+export class ScheduleComponent implements OnInit 
+{
+  public trips: Trip[];
+  public riderTrips: Trip[]=[];
+  public id: number;
+  
+  constructor(private serv: ScheduleService, private auth: AuthService) 
+  { 
+  }
+  ngOnInit() 
+  {
+    this.serv.getTrips().subscribe(
+      data => this.trips=data
+       );
+
+       this.id=+sessionStorage.getItem("userid");
+
+       this.serv.getRiderTrips(this.id).subscribe(
+        data =>{ this.riderTrips=data
+      }
+      );
+  }
+  addTrip(t: Trip)
```

Comment:
For readability, please have empty lines between method declarations, and perhaps some js-doc comments explaining what they are doing,